AssetsPickerViewController/Classes/Album/Controller/AssetsAlbumViewController.swift

```diff
@@ -94,7 +94,7 @@ open class AssetsAlbumViewController: UIViewController {
     open override func loadView() {
         super.loadView()
         view = UIView()
-		view.backgroundColor = .ap_background
+        view.backgroundColor = AssetsManager.shared.pickerConfig.colors.background
```

Comment:
can't you use the `pickerConfig` instance defined on the album controller

Reply:
I've removed the usage of the shared object in the places where I have access to the picker config. 
@alanipce Should I take some time to remove all the usages? I.e for the cells I can quickly create a `configure` method and pass the proper data, please let me know.